node/flags/flags.go

```diff
@@ -238,6 +238,33 @@ var (
 		EnvVar:   common.PrefixEnvVar(EnvVarPrefix, "CHUNK_DOWNLOAD_TIMEOUT"),
 		Value:    20 * time.Second,
 	}
+	DisableDispersalAuthenticationFlag = cli.BoolFlag{
```

Comment:
Why disabling is an option? shouldn't it be non optional?

Reply:
It's enabled by default. Necessary to get things working in inabox. Inabox integration is kind of complex, so I'd like to merge this code with it disabled in the e2e test, and to enable it in a follow up PR.